components/Intro.tsx

```diff
@@ -13,14 +19,14 @@ const Intro = () => (
               height={90}
             />
           </div>
-          <div className="name">{config.author_name}</div>
-          <div className="job">{config.author_position}</div>
+          <div className="name">{author_name}</div>
+          <div className="job">{author_position}</div>
         </div>
         <div className="intro-text">
-          <h1>Hello, I am {config.author_name}</h1>
-          <p style={{ whiteSpace: "break-spaces" }}>
-            {config.site_description}
-          </p>
+          <h1>Hello, I am {author_name}</h1>
+          <span style={{ whiteSpace: "break-spaces" }}>
+            <TinaMarkdown content={site_description} />{" "}
+          </span>
```

Comment:
Do you need a whitespace in here?

Reply:
No, it was added by ESLint and I couldn't remove it, but now I can 🤔